openstudiocore/src/model/AirLoopHVAC.cpp

```diff
@@ -1741,6 +1760,24 @@ void AirLoopHVAC::autosizeDesignSupplyAirFlowRate() {
   getImpl<detail::AirLoopHVAC_Impl>()->autosizeDesignSupplyAirFlowRate();
 }
 
+
+
+/** Presure Drop Adjustments **/  
+double AirLoopHVAC::baselinePressureDropAdjustment() const {
+  return getImpl<detail::AirLoopHVAC_Impl>()->baselinePressureDropAdjustment();
+}
+
+bool AirLoopHVAC::setBaselinePressureDropAdjustment(double baselinePressureDropAdjustment) {
+  getImpl<detail::AirLoopHVAC_Impl>()->setBaselinePressureDropAdjustment(baselinePressureDropAdjustment);
```

Comment:
You have to return a value unless the function is void, C++ does not do the ruby "return the last evaluated thing" thing:

    return getImpl<detail::AirLoopHVAC_Impl>()->setBaselinePressureDropAdjustment(baselinePressureDropAdjustment);